src/Bridge/Laravel/Responses/NoContentApiResponse.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\ApiFormats\Bridge\Laravel\Responses;
+
+/**
+ * This class has for purpose to easily customise status code and/or headers for given content.
```

Comment:
I think we should remove this comment because of a `NoContentApiResponse` shouldn't receive any content :D